resources/platform/linux/README.md

```diff
@@ -0,0 +1,23 @@
+In order to update the libradamsa shared object, clone the radamsa repo
+available at: 
+
+https://gitlab.com/akihe/radamsa
```

Comment:
Include a git clone command, and place in a code section.

Reply:
Done